beefy-gadget/src/lib.rs

```diff
@@ -164,8 +164,13 @@ struct VoteMessage<Hash, Number, Id, Signature> {
 	signature: Signature,
 }
 
+enum BeefyValidator<Id> {
```

Comment:
```suggestion
enum BeefyWorkerMode<Id> {
```

I'd change the naming, cause currently `validator` is duplicated in both variants (and the type itself) and `BeefyValidator::NoValidator` looks a bit weird :)

Reply:
How about changing the type name to `BeefyAuthority`? This would give us `BeefyAuthority::NoValidator` and `BeefyAuthority::Validator`.